pkg/console/controllers/advancedclustermanagement/managedclusters/controller.go

```diff
@@ -517,6 +547,12 @@ func (c *ManagedClusterController) SyncManagedClusterConfigMap(managedClusters [
 			continue
 		}
 
+		controlPlaneTopology, err := managedclusterviewsub.GetInfrastructureConfigCopiedCSVDisabled(olmConfigMCV)
+		if err != nil {
+			klog.V(4).Infof("Error getting copiedCSVDisabled field for managed cluster %v", clusterName)
+			continue
+		}
+
```

Comment:
Don't we need to fetch the infrastructure config MCV as we did for OLMConfig?
```suggestion
		// Check that managed cluster infrastructure config MCV has already been synced, skip if not found
		infraConfigMCV, err := c.dynamicClient.Resource(api.ManagedClusterViewGroupVersionResource).Namespace(clusterName).Get(ctx, api.InfrastructureConfigManagedClusterViewName, metav1.GetOptions{})
		if apierrors.IsNotFound(err) {
			klog.V(4).Infof("Infrastructure config ManagedClusterView not found for managed cluster %v", clusterName)
			continue
		}
		// Skip if unable to get olm config MCV for any other reason
		if err != nil {
			klog.V(4).Infof("Error getting Infrastructure config ManagedClusterView for managed cluster %v", clusterName)
			continue
		}
		controlPlaneTopology, err := managedclusterviewsub.GetInfrastructureConfigControlPlaneTopology(infraConfigMCV)
    if err != nil {
			klog.V(4).Infof("Error getting controlPlaneTopology field for managed cluster %v", clusterName)
			continue
		}
```